docs/prover/checking/coverage-info.md

```diff
@@ -0,0 +1,147 @@
+
+1. Green means that all of the TAC commands that are mapped to the line are in the unsat core, i.e. needed to prove the property. 
+2. Red means that none of the TAC commands that are mapped to the line are in the unsat core, i.e. not needed to prove the property.
+3. Yellow means that some of the TAC commands that are mapped to the line are in the unsat core and some of them are not in the unsat core.
+
+Furthermore, if we have multiple rules/invariants or a parametric rule (such as our `tautology`), we can also have multiple rules/invariants mapping to a single `.sol` or `.spec` line. That is, we generate just a single, joint, visualization for all the rules/invariants (run with `--rule` and `--method` flags to get a visualization for a single rule/method/invariant). And in such case, a yellow line means that some of the commands on the line are needed to prove some of the rules/methods/invariants and some of the commands are not needed. 
+
+The right pane provides detailed information about individual lines and commands mapped to the lines group by the rule/method/invariant name (denoted *rule*) and the *value*, where each *value* refers to a particular `assign`, `assert` or `assume` TAC command that is mapped to the line. For instance, in the Tautology example, we can see that sub-command `other != manager` of line `11` matters (is in the unsat core), whereas `other != e.msg.sender` is not in the unsat core. Also, note that the pane shows e.g. both `other != manager` and `other == manager`; this is because in our TAC representation we encode `other != manager` as `!(other == manager)`. 
+
+
+### Visualization on TAC 
+On contrary to `.sol` and `.spec` visualization, we generate separate TAC unsat core visualization for individual rules/methods/invariants. To access it, click first on the particular rule/method/invariant in the `Rules` pane and then on the `Dump page` button as shown in Figure 3. 
+
+![](tac-visualization-button.png)
+Figure 3: The TAC visualization button.
+
+The visualization here consists only of 2 colors: `green` means that the command is needed and `red` means that the command is not needed (i.e. not in the unsat core). In particular, Figures 4a and 4b show the TAC visualization for the Tautology example. 
+
+![](tautology-tac-a-new.png)
+Figure 4a: The visualization of the Tautology example on TAC, part A. 
+
+![](tautology-tac-b-new.png)
+Figure 4b: The visualization of the Tautology example on TAC, part B.
+
+Namely, only the following TAC commands are in the unsat core:
```

Comment:
perhaps drop the "Namely"

Reply:
Seems like I have not written any academic paper for a while (strict page limits do the trick...) :)